azurelinuxagent/ga/exthandlers.py

```diff
@@ -276,9 +276,6 @@ class ExtHandlersHandler(object):
     def __init__(self, protocol):
         self.protocol = protocol
         self.ext_handlers = None
-        self.last_etag = None
-        self.log_report = False
-        self.log_process = False
```

Comment:
log_report and log_process were not needed

the check for new goal state (self.last_etag) was moved to the UpdateHandler